portfolio/src/main/webapp/style.css

```diff
@@ -158,12 +154,54 @@ padding-left: 25px
   width: 40%;
 }
 
-.effect_text {
+.effect_text{
   margin-right: auto;
 }
 
-.mics_background {
+.map_background{
   padding-top: 10px;
+  padding-bottom: 20px;
+}
+
+#map_game{
+  padding: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+#map_game h2{
+  text-align: center;
+}
+
+
+#map_output_answer {
+  text-align: right;
+}
+
+.map_correct {
+  color: #20CC82;
+}
+
+.map_incorrect {
+  color: red;
+}
+
+#map {
+  margin-top: 10px;
+  padding: 35%; /* 30% ??? */
```

Comment:
nit: Remove the comment once you've tested and decided on a value here.

Reply:
ok